### DISCUSSION TITLE
Naming dokan virtual drives

### DISCUSSION THREAD

**Discussion by Gillous** [dokan-dev/dokany#1022] [Ideas]
Hi!

Is there a way to add a name to the virtual drives created using Mirror? This can be useful when multiple drives are created, and could be implemented with a switch to the command line.

Nice work, by the way!

Cheers!

**Comment by Liryna** [comment]
Hi @Gillous ,
Could you share with a screenshot exactly which name would you like to change ?

**Reply by Gillous** [reply]
Hi!

Thanks for your time!

Here is a screenshot of what I have : 
![2021-10-26 20_18_44-Window](https://user-images.githubusercontent.com/69032162/138920026-5b9b6795-0686-44e0-a962-eed72c13ff37.png)

Drives "U" through "Z" are named "DOKAN". Since I got many different virtual drives, each mapping to a precise content (Documents, Multimedia, Music...), it would be great to name with a switch within the command line, in order to initiate the virtual drive, while knowing what's referring to "U" or "X".

Warm regards, 

Gilles

**Reply by Liryna** [reply]
Oh I see this is for the mirror sample. For now the value is fixed in the mirror code.
I imagine it can easily be done. Do you mind create a feature request issue?

**Reply by Gillous** [reply]
Sure!

👍

**Reply by Liryna** [reply]
For anyone looking at this now. An option was added to customize the name in the latest release.

**Reply by stewnet** [reply]
> For anyone looking at this now. An option was added to customize the name in the latest release.

How? 
`C:\Program Files\Dokan\Dokan Library-2.0.4\sample\mirror: .\mirror.exe /r \\100.90.961.68\root /l m /s /d`
I get DOKAN (M:/)

**Reply by Liryna** [reply]
@stewnet `/v MyVolumeName` https://github.com/dokan-dev/dokany/blob/master/samples/dokan_mirror/mirror.c#L1557
